src/particle/particle/particle.py

```diff
@@ -585,7 +585,7 @@ def load_table(cls, filename=None, append=False, _name=None):
 
     # The following __le__ and __eq__ needed for total ordering (sort, etc)
 
-    def __le__(self, other):
+    def __lt__(self, other):
```

Comment:
Fixed a bug here! The definition was `<`, but the name as `<=`! MyPy expected `__lt__` to make it orderable, so it sort of caught the issue.

Reply:
Cool.